packages/design-picker/src/utils/__tests__/available-designs.test.ts

```diff
@@ -115,24 +119,62 @@ describe( 'Design Picker design utils', () => {
 			const mockDesign = availableDesignsConfig.featured[ 0 ];
 			const mockDesignPremium = availableDesignsConfig.featured[ 1 ];
 			const mockDesignAlpha = availableDesignsConfig.featured[ 3 ];
-			expect( getAvailableDesigns( true, false ) ).toEqual( {
-				featured: [ mockDesign, mockDesignPremium, mockDesignAlpha ],
-			} );
+			expect( getAvailableDesigns( { includeAlphaDesigns: true, useFseDesigns: false } ) ).toEqual(
+				{
+					featured: [ mockDesign, mockDesignPremium, mockDesignAlpha ],
+				}
+			);
 		} );
 
 		it( 'should get only FSE, non-alpha designs', () => {
 			const mockDesignFSE = availableDesignsConfig.featured[ 2 ];
-			expect( getAvailableDesigns( false, true ) ).toEqual( {
-				featured: [ mockDesignFSE ],
-			} );
+			expect( getAvailableDesigns( { includeAlphaDesigns: false, useFseDesigns: true } ) ).toEqual(
+				{
+					featured: [ mockDesignFSE ],
+				}
+			);
 		} );
 
 		it( 'should get all non-alpha, non-FSE designs', () => {
 			const mockDesign = availableDesignsConfig.featured[ 0 ];
 			const mockDesignPremium = availableDesignsConfig.featured[ 1 ];
-			expect( getAvailableDesigns( false, false ) ).toEqual( {
-				featured: [ mockDesign, mockDesignPremium ],
-			} );
+			expect( getAvailableDesigns( { includeAlphaDesigns: false, useFseDesigns: false } ) ).toEqual(
+				{
+					featured: [ mockDesign, mockDesignPremium ],
+				}
+			);
+		} );
+
+		it( 'should randomize the results order when the randomize flag is specified', () => {
```

Comment:
Not the test unit test ever written, but hopefully it does the job

Reply:
Maybe a better approach at the design-picker level would be to extract the shuffle so you can mock it and ensure that it's called?

Then it looks like you can test the implementation itself by mocking `global.Math.random`. Stack Overflow pointed out the `jest-mock-random` package, but it boils down to
`const originalMathRandom = Math.random; Math.random = require('lodash').constant(0.7)`

Reply:
Those are definitely good ideas! I'm going to track this in a separate follow-up issue https://github.com/Automattic/wp-calypso/issues/51743

Reply:
This test seems to be flaky. It has failed a few times in `trunk`:

https://teamcity.a8c.com/buildConfiguration/calypso_RunAllUnitTests/6161052?showLog=6161052_27701_26895.27686.27701

https://teamcity.a8c.com/buildConfiguration/calypso_RunAllUnitTests/6160981?showLog=6160981_27226_27003.27226

Would it be possible to disable it until it is fixed?

Reply:
#52634 disables the test